casepro/backend/rapidpro.py

```diff
@@ -330,8 +332,14 @@ def push_outgoing(self, org, outgoing, as_broadcast=False):
                 msg.backend_broadcast_id = broadcast.id
                 msg.save(update_fields=("backend_broadcast_id",))
 
-    def push_contact(self, org, contact):
-        return
+    def push_contact(self, org, normalised_urn):
```

Comment:
`normalized_urn` for consistency

Reply:
let's call this method `resolve_urn` as well since it's no longer pushing a contact